src/ledger/events.rs

```diff
@@ -0,0 +1,95 @@
+use chrono::DateTime;
+use chrono::Utc;
+use ethereum_types::U256;
+use revm::primitives::Address;
+use uuid::Uuid;
+
+use crate::eth::primitives::BlockNumber;
+use crate::eth::primitives::Hash;
+
+/// Represents token transfers (debits and credits) associated with a specific Ethereum account within a single transaction.
+///
+/// The `account_address` field identifies the main account involved in these transfers.
+///
+/// An event will be generated for each account involved in a transaction, meaning if a transaction is not the main account in this event,
+/// in another event it will be and credit and debit parties will be adjusted accordingly.
+pub struct AccountTransfersEvent {
+    /// ID of the event publication.
+    ///
+    /// If the event is republished, a new ID will be generated for the publication.
+    ///
+    /// Format: UUID v7 (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx)
+    pub publication_id: Uuid,
+
+    /// Datetime of the event publication.
+    ///
+    /// Format: ISO 8601
+    pub publication_datetime: DateTime<Utc>,
```

Comment:
Should we go with ISO 8601 or UNIX timestamps for datetimes?
I think ISO 8601 is more readable.

Reply:
I think ISO is better as well. It is easier to understand and no need to convert anything. We want to facilitate the consumers of these events, which will be ourselves too in case of debugging.

Reply:
I prefer ISO 8601 as well, but if message size is a constraint I believe unix timestamp is better